### PR TITLE
Adds the catghost to donator's special ghosts

### PR DESCRIPTION
**Pull request by Niknakflak**
Adds this little cute sprite for byond donators 
image

Sprite by me
Originally made way back for TehZombehz

I made a fix for the partybutton if a fix is required.

### CODE REVIEW

**Comment by Niknakflak** [comment]
Stand by for defucking or PR

File: code/modules/client/preferences_toggles.dm [hunk 1]
```diff
@@ -235,7 +235,7 @@ var/global/list/ghost_forms = list("ghost","ghostking","ghostian2","skeleghost",
 							"ghost_blue","ghost_yellow","ghost_green","ghost_pink", \
 							"ghost_cyan","ghost_dblue","ghost_dred","ghost_dgreen", \
 							"ghost_dcyan","ghost_grey","ghost_dyellow","ghost_dpink", "ghost_purpleswirl","ghost_funkypurp","ghost_pinksherbert","ghost_blazeit",\
-							"ghost_mellow","ghost_rainbow","ghost_camo","ghost_fire")
+							"ghost_mellow","ghost_rainbow","ghost_camo","ghost_fire", "catghost")
```
Comment (KorPhaeron) [inline code/modules/client/preferences_toggles.dm]: Different naming convention than the others this is triggering

Comment (Niknakflak) [inline code/modules/client/preferences_toggles.dm]: You'll get over it

**Comment by Chocobro** [comment]
:3

**Pull request merged**